web-app/src/app/utils/config.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Get environment config
+ * @param key variable key
+ * @returns variable value or empty string
+ */
+export const getEnvConfig = (key: string): string => {
+  return process.env[key] ?? '';
+};
```

Comment:
Could you provide guidance on how the key should be setup for local dev?

Reply:
Added a section on the README file on how to provide the values; also updated the outdated script section.